### PR TITLE
Add return type to __toString()

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 6.0
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

Allowed by https://wiki.php.net/rfc/stringable

### CODE REVIEW

**Review by yceruto — APPROVED** [review]

**Review by StaffNowa — APPROVED** [review]
Good job 👌

**Comment by derrabus** [comment]
Okay, wow. https://3v4l.org/dTIpF ❤️

**Review by derrabus — APPROVED** [review]

File: src/Symfony/Component/Config/Resource/ResourceInterface.php [hunk 1]
```diff
@@ -26,8 +26,6 @@ interface ResourceInterface
      * to be identical for different ResourceInterface instances referring to the same
      * resource; and it should be unlikely to collide with that of other, unrelated
      * resource instances.
-     *
-     * @return string A string representation unique to the underlying Resource
      */
-    public function __toString();
+    public function __toString(): string;
```
Comment (derrabus) [inline src/Symfony/Component/Config/Resource/ResourceInterface.php:30]: Couldn't we remove that method completely by declaring…
```php
interface ResourceInterface extends Stringable
```

Comment (nicolas-grekas) [reply src/Symfony/Component/Config/Resource/ResourceInterface.php:30]: We prefer keeping the comment, don't we?

Comment (derrabus) [reply src/Symfony/Component/Config/Resource/ResourceInterface.php:30]: Fair enough.

**Comment by derrabus** [comment]
Thank you Nicolas.

**Pull request merged**